packages/server/package.json

```diff
@@ -43,6 +51,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "28.1.3",
     "prettier": "^2.3.2",
+    "prisma": "latest",
```

Comment:
best to pin this to a version, cant guarantee we will be updating to latest if there are breaking changes in a version